### PR TITLE
Remove model-approval-status read-only  from generator.yaml

### PR DESCRIPTION
**Pull request by ryansteakley**
Description of changes:
Remove model-approval-status as a read-only field it is not a status field, it is a part of the spec, don't need it in both places

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: generator.yaml [hunk 1]
```diff
@@ -494,11 +494,6 @@ resources:
         from:
           operation: DescribeModelPackage
           path: ModelPackageStatusDetails
-      ModelApprovalStatus:
```
Comment (surajkota) [inline generator.yaml:497]: something wrong here, it undoes the late initializer stuff?

Comment (ryansteakley) [reply generator.yaml:497]: Opened pr before late_initialize got merged, will rebase

**Pull request merged**